### ISSUE TITLE
CONV2 occurrence mismatch in timeline

### ISSUE THREAD

**Issue by benzhang1986** [tensorflow/tensorflow#23447]
**System information**
- Linux Ubuntu 16.04
- TensorFlow installed binary
- TensorFlow 1.10 and 1.12 both
- Python 2.7:
- CUDA 9.0
- cuDNN 7.3
- GPU model and memory: NVIDIA V100 (16GB) x 8

**Describe the current behavior**
dump timeline of benchmark with following command:
python tf_cnn_benchmarks.py --local_parameter_device=gpu --model=resnet50 --data_dir=/home/nvme/dataset/imagenet/tf_records/train/ --variable_update=replicated --all_reduce_spec=nccl --use_fp16=True --batch_size=128 --num_gpus=1 --trace_file=/home/timeline/benchmark_resnet50_imagenet_gpu1.json

then the number of CONV2D op send to GPU is 53
![image](https://user-images.githubusercontent.com/15809856/47891157-fc087300-de8c-11e8-81d6-40446b20ec3d.png)

but GPU run so many CONV2D:
![image](https://user-images.githubusercontent.com/15809856/47891199-31ad5c00-de8d-11e8-8c89-3823de6d5ecc.png)

I think they should be 53 both.

**Comment by harshini-gadige** [comment]
@zheng-xq   PTAL

**Comment by tensorflowbutler** [comment]
Nagging Assignee @yongtang: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tilakrayal** [comment]
@benzhang1986,
Is this still an issue?
Could you please update TensorFlow to the latest stable version v.2.9 and let us know if you are facing the same error. Thank you!
